pom.xml

```diff
@@ -71,8 +71,8 @@
 
   <properties>
     <!--Dependency versions-->
-    <akka.version>2.4.2</akka.version>
-    <akka.http.version>2.4.2</akka.http.version>
+    <akka.version>2.4.11.1</akka.version>
```

Comment:
Perhaps I'm crazy here, but this doesn't seem like a valid Akka version.  It also isn't the latest (2.4.17).  What's the deal?